pkg/flexvolume/client.go

```diff
@@ -9,6 +12,20 @@ type client struct {
 	apiClient APIClient
 }
 
+type AttachSuccessOutput struct {
```

Comment:
Yes kubernetes flexvolume util watches the stdout when it calls the flexvolume plugin's commands. They expect a json output with pre-defined key value pairs in it.